src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -619,6 +619,13 @@
     "NAME": "Toil",
     "DESCRIPTION": "Deal !D! damage. NL If this kills an enemy, permanently upgrade a random card. NL Exhaust."
   },
+  "stsjorbsmod:Toll": {
+    "NAME": "Toll",
+    "DESCRIPTION": "Deal damage equal to !M! times the sum of your relic counters.",
```

Comment:
Probably simpler for the unupgraded version to just be `"Deal damage equal to the sum of your relic counters"` and leave the `!M! times` for the upgraded description; I appreciate the attempt to be more localization friendly, but player friendliness is more important.